mhkit/wave/resource.py

```diff
@@ -158,19 +157,19 @@ def jonswap_spectrum(f, Tp, Hs, gamma=None):
     Gf[hind] = gamma**np.exp(-(f[hind]-fp)**2/(2*sigb**2*fp**2))
     C = 1- 0.287*np.log(gamma)
     Sf = C*S_f*Gf
-    
+
     col_name = 'JONSWAP ('+str(Hs)+'m,'+str(Tp)+'s)'
-    S = pd.DataFrame(Sf, index=f, columns=[col_name])  
-    
+    S = pd.DataFrame(Sf, index=f, columns=[col_name])
+
     return S
 
 ### Metrics
-def surface_elevation(S, time_index, seed=123, frequency_bins=None,phases=None):
+def surface_elevation(S, time_index, seed=None, frequency_bins=None,phases=None):
```

Comment:
The changes to this function are causing failures in the test. We should discuss what the correct algorithms are for computing the surface elevation.

Reply:
I will create a separate PR to address  this function. It has bugs in it and is very slow. My guess is the tests fail because of the seed not being hardcoded to 123. Or due to the truncation (bug). I will make sure to fix the tests as  well in that PR.